packages/navigator/src/components/Navbar.tsx

```diff
@@ -22,209 +28,213 @@ interface INavbarProps {
+        onResize()
+        window.addEventListener('resize', onResize)
 
-    if (props.theme === 'Cupertino') {
-      onResize()
-      window.addEventListener('resize', onResize)
+        return () => {
+          window.removeEventListener('resize', onResize)
+        }
+      }
+    }, [screenHelmetProps])
+
+    const onBackClick = () => {
+      pop()
+    }
 
+    const closeButton =
+      props.onClose &&
+      props.isRoot &&
+      !screenHelmetProps.noCloseButton &&
+      (screenHelmetProps.customCloseButton ? (
+        <a
+          className={css.closeButton}
+          role="text"
+          aria-label={props.closeButtonAriaLabel}
+          onClick={props.onClose}
+        >
+          {screenHelmetProps.customCloseButton}
+        </a>
+      ) : (
+        <a
+          className={css.closeButton}
+          role="text"
+          aria-label={props.closeButtonAriaLabel}
+          onClick={props.onClose}
+        >
+          <IconClose className={css.svgIcon} />
+        </a>
+      ))
+
+    const backButton =
+      !props.isRoot &&
+      !screenHelmetProps.noBackButton &&
+      (screenHelmetProps.customBackButton ? (
+        <a
+          className={css.backButton}
+          role="text"
+          aria-label={props.backButtonAriaLabel}
+          onClick={onBackClick}
+        >
+          {screenHelmetProps.customBackButton}
+        </a>
+      ) : (
+        <a
+          className={css.backButton}
+          role="text"
+          aria-label={props.backButtonAriaLabel}
+          onClick={onBackClick}
+        >
+          {props.theme === 'Cupertino' && props.isPresent ? (
+            <IconClose className={css.svgIcon} />
+          ) : (
+            <IconBack className={css.svgIcon} />
+          )}
+        </a>
+      ))
+
+    const isLeft = !!(
+      (screenHelmetProps.closeButtonLocation === 'left' && closeButton) ||
+      backButton ||
+      screenHelmetProps.appendLeft
+    )
+
+    const noBorder = screenHelmetProps.noBorder
+
+    const { lifecycleHooks } = usePlugins()
+
+    const onMountNavbar = useCallback(() => {
+      lifecycleHooks.forEach((hook) => {
+        const context = {
+          screenHelmetProps,
+        } as any
+        hook?.onMountNavbar?.(context)
+      })
+    }, [lifecycleHooks, screenHelmetProps])
+
+    const onUnmountNavbar = useCallback(() => {
+      lifecycleHooks.forEach((hook) => {
+        const context = {
+          screenHelmetProps,
+        } as any
+        hook?.onUnmountNavbar?.(context)
+      })
+    }, [lifecycleHooks, screenHelmetProps])
+
+    useEffect(() => {
+      onMountNavbar()
       return () => {
-        window.removeEventListener('resize', onResize)
+        onUnmountNavbar()
       }
-    }
-  }, [screenHelmetProps])
+    }, [onMountNavbar, onUnmountNavbar])
 
-  const onBackClick = () => {
-    pop()
-  }
+    const mounted = useMounted({ afterTick: true })
+    useImperativeHandle(ref, () => ({ mounted }), [mounted])
```

Comment:
```suggestions
    useImperativeHandle(ref, () => ({ 
      mounted: () => mounted,
      navbar: () => navbarRef.current,
    }), [mounted])
```

`Card`에서 또 navbar.current로 잡아서 하는것보다는 그냥 여기서 같이 빼는게 어떨까요?
`useImperativeHandle`의 쓰는 이유이기도 해서 이렇게 빼면 더 좋을 것 같은뎅